Source/Editor/Modules/UIModule.cs

```diff
@@ -574,6 +575,11 @@ private void OpenProject()
             }
         }
 
+        private void OpenProjectDirectory()
+        {
+            FileSystem.ShowFileExplorer(Editor.Instance.GameProject.ProjectFolderPath);
```

Comment:
Instead of using `Editor.Instance` you should use the read only `Editor` field from the inherited `EditorModule` class.